packages/web/src/components/data-entry/FormTile.js

```diff
@@ -306,30 +376,45 @@ const BasicForm = (props) => {
 }
 
 const AdvancedForm = (props) => {
-  let unlistedState
-  let unlistedButtonTitle
-  const showUnlisted = props.type === 'track' && props.showUnlistedToggle
-  if (showUnlisted) {
-    unlistedState = {
+  let availabilityButtonTitle
+  let availabilityState = {
+    is_premium: props.defaultFields.is_premium,
+    premium_conditions: props.defaultFields.premium_conditions
+  }
+  const showAvailability = props.type === 'track' && props.showUnlistedToggle
+  if (showAvailability) {
+    availabilityState = {
+      ...availabilityState,
       unlisted: props.defaultFields.is_unlisted,
       genre: props.defaultFields.field_visibility.genre,
       mood: props.defaultFields.field_visibility.mood,
       tags: props.defaultFields.field_visibility.tags,
       share: props.defaultFields.field_visibility.share,
       plays: props.defaultFields.field_visibility.play_count
     }
-    unlistedButtonTitle = unlistedState.unlisted
-      ? messages.hidden
-      : messages.public
+
+    availabilityButtonTitle = messages.public
+    if (availabilityState.unlisted) {
+      availabilityButtonTitle = messages.hidden
+    } else if (availabilityState.is_premium) {
+      if (
+        availabilityState.premium_conditions &&
+        'nft_collection' in availabilityState.premium_conditions
+      ) {
+        availabilityButtonTitle = messages.collectibleGated
+      } else {
+        availabilityButtonTitle = messages.specialAccess
+      }
+    }
   }
 
-  const [isUnlistedModalOpen, setIsUnlistedModalOpen] = useState(false)
+  const [isAvailabilityModalOpen, setIsAvailabilityModalOpen] = useState(false)
   const [hideRemixes, setHideRemixes] = useState(
     !(props.defaultFields?.field_visibility?.remixes ?? true)
   )
 
   // Need to update two fields in the metadata.
-  const didUpdateUnlistedState = (newState) => {
+  const didUpdateAvailabilityState = (newState) => {
```

Comment:
nit: I'm a little worried about "availability" being the term of choice here, since I think it may be overloaded from trusted notifier/content availability